src/makielayout/lineaxis.jl

```diff
@@ -581,18 +581,20 @@ function get_ticks(tickfunction::Function, any_scale, formatter, vmin, vmax)
     return tickvalues, ticklabels
 end
 
+_logbase(s::ReversibleScale) = s.logbase
 _logbase(::typeof(log10)) = "10"
 _logbase(::typeof(log2)) = "2"
 _logbase(::typeof(log)) = "e"
+_logbase(::Any) = ""
 
-
-function get_ticks(::Automatic, scale::Union{typeof(log10), typeof(log2), typeof(log)},
-        any_formatter, vmin, vmax)
-    get_ticks(LogTicks(WilkinsonTicks(5, k_min = 3)), scale, any_formatter, vmin, vmax)
+function get_ticks(::Automatic, scale::Union{LogFunctions,ReversibleScale}, any_formatter, vmin, vmax)
+    wt = WilkinsonTicks(5, k_min = 3)
+    ticks = isempty(_logbase(scale)) ? wt : LogTicks(wt)
```

Comment:
are you using `_logbase` for nothing else than this `isempty` check? I don't think we really need it, log functions should just dispatch to LogTicks and all others don't

Reply:
No, I'm not. I will change it to a `nothing` check instead then.